src/controllers.php

```diff
@@ -8,11 +8,15 @@
 
 //Request::setTrustedProxies(array('127.0.0.1'));
 
-$app->get('/', function () use ($app) {
-    return $app['twig']->render('index.html.twig', array());
+$app->get('/index', function () use ($app) {
+    return $app['twig']->render('index.html.twig', []);
 })
-->bind('homepage')
-;
+->bind('index');
+
+$app->get('/login', function () use ($app) {
+    return $app['twig']->render('login.html.twig', []);
```

Comment:
funkcja render powinna domyslnie miec ustalone ze drugi argument to [] wiec jak nie ma potrzeby recznie napisywac domyslna wartosc to mozna go pominac